repos/envelope.go

```diff
@@ -0,0 +1,93 @@
+package repos
+
+import (
+	"fmt"
+	"github.com/go-pg/pg/v10"
+	"math/rand"
+	// "time"
+	// "msudenver.edu/ledger/db"
+	// "msudenver.edu/ledger/repos"
+)
+
+type EnvelopeRepo struct {
+	db *pg.DB
+}
+
+// Basic struct for "Goals" & "Expenses" envelope'ing (From Ledger proposal)
+type Envelope struct {
+	ID             int64  `json:"-"`
+	Name           string `pg:",unique,notnull"`
+	CurrentBalance int
+	StartDate      string `pg:",notnull"`
+	TargetDate     string `pg:",notnull"`
```

Comment:
Dates should be `time.Time` (see https://github.com/rooftopsparrow/ledger/blob/master/repos/account.go#L26)